### PR TITLE
Move quota protocol to session

### PR DESCRIPTION
**Pull request by jacobhaven**
r? @maxtaco

### CODE REVIEW

File: protocol/Makefile [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+	npm i
+$(AVDLC):
 	npm i
 
 # By default, all avdl/*.avdl are considered and made into outputs
 # as long as they have messages in them. BUT you can override this
 # check via -f. So the two files specified here via -f don't have
 # messages but should be made into .json's...
-build-stamp: avdl/*.avdl $(AVDL2JSON) | config
+build-stamp: avdl/*.avdl | $(AVDL2JSON)
```
Comment (maxtaco) [inline protocol/Makefile]: can you explain this change?

**Comment by maxtaco** [comment]
had one question....

**Comment by maxtaco** [comment]
oh, this change has broken the build^

**Comment by maxtaco** [comment]
Ok, i think it makes sense just to delete this protocol message.

File: protocol/Makefile (hunk 1 above)
Comment (jacobhaven) [inline protocol/Makefile]: Changing to a non-phony dependency means that `npm i` won't be called if `node_modules/.bin/avdl2json` (or `node_modules/.bin/avdlc`) already exists.

Comment (maxtaco) [inline protocol/Makefile]: ok cool

**Pull request closed without merging**